client/state/plans/test/selectors.js

```diff
@@ -113,4 +117,33 @@ describe( 'selectors', () => {
 			expect( price ).to.eql( null );
 		} );
 	} );
+
+	describe( '#getPlanSlug()', () => {
+		it( 'should return plan product_slug', () => {
+			const state = getStateInstance();
```

Comment:
Could we maybe avoid using the fixture in these new tests? It makes it easier to read these test cases, even if we end up copying and pasting more code in tests (which is OK).